src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -3010,4 +3012,51 @@ class Organization: ADOSVTBase
         return $controlResult
     }
 
+    hidden [ControlResult] CheckCreatePermissionsForFeed ([ControlResult] $controlResult) {
+
+        try {
+            $controlResult.VerificationResult = [VerificationResult]::Failed
+            $roleAssignments = @();
+
+            #Fetch feeds RBAC
+            if ($this.FeedGlobalPermissions.Count -eq 0) {
+                $url = "https://feeds.dev.azure.com/$($this.OrganizationContext.OrganizationName)/_apis/Packaging/GlobalPermissions?includeIds=true&api-version=5.1-preview.1"
+                $this.FeedGlobalPermissions = @([WebRequestHelper]::InvokeGetWebRequest($url));
+            }
+
+            #get identity details for groups fetched from above api
+            $responseObj = @($this.FeedGlobalPermissions | where-object {$_.role -eq 'feedCreator'})
+            if ($responseObj.Count -gt 0)
+            {
+                $ids = $responseObj.identityId -join ';'
+                $url = "https://dev.azure.com/$($this.OrganizationContext.OrganizationName)/_apis/IdentityPicker/Identities?api-version=5.0-preview.1" #-f $($OrgName), $($groupObj.entityId)
+                $body = '{"query":"'+ $ids +'","identityTypes":["group"],"operationScopes":["ims"],"queryTypeHint":"uid","properties":["DisplayName","ScopeName"]}'
+                $response = Invoke-WebRequest -Uri $url -Method Post -ContentType "application/json" -Body $body -UseBasicParsing
+            
+                $groups = $response.Content | Convertfrom-json
+                $roleAssignments = $groups.results.identities.displayname
+
+                # Checking if everyone in the organization has create permission on feed
+                $restrictedGroups = @($roleAssignments | Where-Object {"Project Collection Valid Users" -contains $_.split('\')[-1] })
+                $restrictedGroupsCount = $restrictedGroups.Count
+
+                # fail the control if everyone in the organization has create permission on feed
+                if ($restrictedGroupsCount -gt 0) {
+                    $controlResult.AddMessage([VerificationResult]::Failed, "Feeds create permission has been granted to everyone in the organization.");
+                }
+                else {
+                    $controlResult.AddMessage([VerificationResult]::Passed, "Feeds create permission has not been granted to everyone in the organization.");
+                }
+            }
+            else
+            {
+                $controlResult.AddMessage([VerificationResult]::Passed, "Feeds create permission has not been granted to everyone in the organization.");
```

Comment:
Can we keep status as Verify instead of Passed?  We are not sure whether  of permissions here